packages/astro-ghostcms/README.md

```diff
@@ -59,7 +59,31 @@ import GhostCMS from '@matthiesenxyz/astro-ghostcms';
 // https://astro.build/config
 export default defineConfig({
   site: "https://YOUR-DOMAIN-HERE.com"
-  integrations: [GhostCMS()],
+  integrations: [
+    GhostCMS({
+      // Config Options
+      ghostURL: "http://example.com", // Recommended to set here, Can also set in .env as CONTENT_API_URL
+      ThemeProvider: { // Allows you to pass config options to our ThemeProvider if enabled.
+        disableThemeProvider: false, // OPTIONAL - Default False
+        theme: "@matthiesenxyz/astro-ghostcms-theme-default", // OPTIONAL - Default Theme shown.
+      };
+      disableDefault404: false, // Allows the user to disable the default `/404 page, to be able to create their own under `/src/pages/404.astro`.
+      enableRSSFeed: true, // Allows the user to Enable or disable RSS Feed Generation. Default: true
+      enableOGImages: true, // Allows the user to Enable or disable OG Image Generation. Default: true
+      fullConsoleLogs: false, // Show the full Log output from All parts of Astro-GhostCMS
```

Comment:
This could be called something like `verbose` or `logLevel` or something. Most devs would search for `verbose` for verbose logging

Reply:
updated to verbose... i really don't know why i didn't call it that from the first place XD